Sources/UInt128.swift

```diff
@@ -147,18 +115,9 @@ public struct UInt128 {
             throw UInt128Errors.emptyString
         }
         // Internal variables.
-        let radix: UInt8
+        let radix: UInt8 = string.radix
```

Comment:
I don't believe they are. Anything that imports UInt128 will have those extensions available on their String type.